service/src/main/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -203,17 +204,27 @@ private PolicyEvaluationResult evalPolicySet(final PolicySet policySet, final St
             }
             result = new PolicyEvaluationResult(effect, new ArrayList<>(subjectAttributes),
                     new ArrayList<>(resourceAttributes), resolvedResourceUris);
+        } catch (AttributeTraversalException ae) {
+            result = handlePolicyEvaluationException(policySet, subjectIdentifier, resourceURI, ae);
         } catch (Throwable e) {
-            StringBuilder builder = new StringBuilder();
-            builder.append("Exception occured while evaluating the policy set. Policy Set ID:")
-                    .append(policySet.getName()).append(" subject:").append(subjectIdentifier)
-                    .append(", Resource URI: ").append(resourceURI);
-            LOGGER.error(builder.toString(), e);
-            result = new PolicyEvaluationResult(Effect.INDETERMINATE);
+            result = handlePolicyEvaluationException(policySet, subjectIdentifier, resourceURI, e);
         }
         return result;
     }
 
+    private PolicyEvaluationResult handlePolicyEvaluationException(final PolicySet policySet,
+            final String subjectIdentifier, final String resourceURI, final Throwable e) {
+        PolicyEvaluationResult result = new PolicyEvaluationResult(Effect.INDETERMINATE);
+        StringBuilder logMessage = new StringBuilder();
+        logMessage.append("Exception occured while evaluating the policy set. Policy Set ID:")
```

Comment:
I think you need a space after the semicolon here. Also consider single quoting the values you are injecting into the string. Makes it easier to read and select the text.